privatePackages/assert-project/test/index.ts

```diff
@@ -26,3 +26,12 @@ test('assertProject() store functions', async (t) => {
 
   t.end()
 })
+
+test('direct storeHas() without project', async (t) => {
+  const project = assertProject(t, path.join(__dirname, 'fixture/project'), 'registry.npmjs.org')
+
+  const storePath = path.join(__dirname, 'fixture/store/2')
+  await project.storeHas('is-positive', '3.1.0', storePath)
```

Comment:
this function is completely unrelated to `project` if the store is not the project's store.

So I think it should be a different util function. Maybe just `storeHas(storePath, pkgName, pkgVersion)`

Or something like `assertStore()` (similarly to `assertProject`). And then you can use it as `await store.has()`...

Reply:
Ok, that makes the most sense. I was originally confused because I thought that `assert-project` was in charge of creating the store. Taking a deeper dive into the source code, I see that is not the case.

I can make a new `assert-store` privatePackage.